packages/client/screens/UserProfile/UserFollowersSearch/__tests__/UserFollowersSearch.test.tsx

```diff
@@ -0,0 +1,719 @@
+                                        ).toBeTruthy();
+                                    });
+                                },
+                            },
+                        },
+
+                        'pressed an other user card': {
+                            meta: {
+                                test: async ({ screen }: TestingContext) => {
+                                    invariant(
+                                        screen !== undefined,
+                                        'Screen must have been rendered before being in this state',
+                                    );
+
+                                    await waitFor(() => {
+                                        expect(
+                                            screen.getByTestId(
+                                                'default-profile-page-screen',
+                                            ),
+                                        ).toBeTruthy();
+                                    });
+                                },
+                            },
+                        },
+                    },
+
+                    on: {
+                        'Load more followers results': {
+                            target: '.second page is loaded on screen',
+                        },
+
+                        'fill searched user nickname query and submit': {
+                            target: '.filtered users list',
+                        },
+
+                        'fill my nickname and submit': {
+                            target: '.filtered users list',
+                        },
+
+                        'cancel search bar': {
+                            target: '.first page is loaded on screen',
+                        },
+
+                        'press my user card': {
+                            target: '.pressed my user card',
+                        },
+
+                        'press an other user card': {
+                            target: '.pressed an other user card',
+                        },
+                    },
+                },
+
+                'User relations are forbidden': {
+                    meta: {
+                        test: async ({ screen }: TestingContext) => {
+                            invariant(
+                                screen !== undefined,
+                                'Screen must have been rendered before being in this state',
+                            );
+
+                            await waitFor(() => {
+                                const forbiddenLabel =
+                                    screen.getByText(/followers.*forbidden/i);
+                                expect(forbiddenLabel).toBeTruthy();
+                            });
+                        },
+                    },
+                },
+            },
+        },
+    },
+});
+
+//User events
+const searchUserFollowersTestModel = createTestModel<TestingContext>(
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏This is a really good first test model!

Reply:
:pray: